src/test/java/jalau/at18/Azul/BagSenderTest.java

```diff
@@ -0,0 +1,36 @@
+package jalau.at18.azul;
+
+import static org.junit.Assert.assertNotEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+public class BagSenderTest {
+    
+    @Test
+
+    public void shouldGetBag (){
+        BagSender bag = new BagSender();
+        List<Tile> testbag = new ArrayList<>();
+        testbag.add(Tile.RED);
+        testbag.add(Tile.RED);
+        testbag.add(Tile.RED);
+        testbag.add(Tile.RED);
+        bag.saveBag();
+        assertNotEquals(testbag, bag.saveRemainingBag());
+    }
+    
+    /*@Test
+    public void shouldGetFactoryGroup (){
+        BagSender bag = new BagSender();
+        List<Tile> testbag = new ArrayList<>();
+        testbag.add(Tile.RED);
+        testbag.add(Tile.RED);
+        testbag.add(Tile.RED);
+        testbag.add(Tile.RED);
+        bag.saveBag();
+        assertNotEquals(testbag, bag.getFactoryGroup());
```

Comment:
Erase the comment of the code if it's no more needed :)

Reply:
This code haas been removed, Thanks Dani for addressing this issue